ui/src/polyfills.ts

```diff
@@ -32,21 +31,21 @@ import 'core-js/es6/string';
 import 'core-js/es6/array';
 // import 'core-js/es6/regexp';
 // import 'core-js/es6/map';
+// import 'core-js/es6/weak-map';
```

Comment:
They are part of the Angular CLI files. Thus I would rather leave the Angular provided files as unmodified as possible.  That way we know also all the relevant polyfills and can enabled them as needed we start testing across more browsers.